src/anders_mathiesen_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,26 @@
+def sort_list(list_data):
+    new_data = list(list_data)
+    for number in range(len(list_data)):
+        if number == len(list_data) - 1:
+            break
+        first_number = new_data[number]
+        second_number = new_data[number + 1]
+        if first_number > second_number:
+            new_data[number] = second_number
+            new_data[number + 1] = first_number
+    return new_data
+
+
+def bubble_sort(random_list):
+    for _ in range(len(random_list)):
+        random_list = sort_list(random_list)
```

Comment:
This implementation is confusing and inefficient. The `sort_list()` function doesn't actually sort the list, and it doesn't take advantage of the fact the the last `n` elements of the list are sorted at the `n`th iteration.